components/eam/src/physics/cam/zm_conv_intr.F90

```diff
@@ -54,249 +49,210 @@ module zm_conv_intr
    logical  ::    convproc_do_gas 
    logical  ::    clim_modal_aero
 
-!=========================================================================================
 contains
-!=========================================================================================
-
-subroutine zm_conv_register
 
-!----------------------------------------
-! Purpose: register fields with the physics buffer
-!----------------------------------------
+!-------------------------------------------------------------------------------
+!-------------------------------------------------------------------------------
 
-  use physics_buffer, only : pbuf_add_field, dtype_r8
-
-  implicit none
+subroutine zm_conv_register
+   !----------------------------------------------------------------------------
+   ! Purpose: register fields with the physics buffer
+   !----------------------------------------------------------------------------
+   use physics_buffer, only : pbuf_add_field, dtype_r8
+   implicit none
 
-  integer idx
+   integer idx
 
-! Flux of precipitation from deep convection (kg/m2/s)
+   ! Flux of precipitation from deep convection (kg/m2/s)
    call pbuf_add_field('DP_FLXPRC','global',dtype_r8,(/pcols,pverp/),dp_flxprc_idx) 
 
-! Flux of snow from deep convection (kg/m2/s) 
+   ! Flux of snow from deep convection (kg/m2/s) 
    call pbuf_add_field('DP_FLXSNW','global',dtype_r8,(/pcols,pverp/),dp_flxsnw_idx) 
 
-! deep gbm cloud liquid water (kg/kg)
+   ! deep gbm cloud liquid water (kg/kg)
    call pbuf_add_field('DP_CLDLIQ','global',dtype_r8,(/pcols,pver/), dp_cldliq_idx)  
 
-! deep gbm cloud liquid water (kg/kg)    
+   ! deep gbm cloud liquid water (kg/kg)    
    call pbuf_add_field('DP_CLDICE','global',dtype_r8,(/pcols,pver/), dp_cldice_idx)  
 
-
-! DCAPE-UPL
-
+   ! DCAPE-UPL
    if (trigdcape_ull .or. trig_dcape_only) then
-    ! temperature from physics in n-1 time step
-    call pbuf_add_field('T_STAR','global',dtype_r8,(/pcols,pver/), t_star_idx)
-    ! moisturetendency from physics in n-1 time step 
-    call pbuf_add_field('Q_STAR','global',dtype_r8,(/pcols,pver/), q_star_idx)
+      ! temperature from physics in n-1 time step
+      call pbuf_add_field('T_STAR','global',dtype_r8,(/pcols,pver/), t_star_idx)
+      ! moisturetendency from physics in n-1 time step 
```

Comment:
Preexisting inaccurate comment.  it is certainly not tendency.

Reply:
ah right, I'll fix this.